pkg/gardenlet/apis/config/types.go

```diff
@@ -221,6 +221,12 @@ type ShootControllerConfiguration struct {
 	// DNSEntryTTLSeconds is the TTL in seconds that is being used for DNS entries when reconciling shoots.
 	// Default: 120s
 	DNSEntryTTLSeconds *int64
+	// LeaseDurationSeconds defines how long the lease is valid (used for Lease.spec.leaseDurationSeconds).
+	//Default : 2s
+	LeaseDurationSeconds *int32
```

Comment:
Why are these fields added to the `ShootControllerConfiguration`? Shouldn't it rather be the `SeedControllerConfiguration`?

Reply:
Moved the fields to `SeedControllerConfiguration` .